snuba/datasets/events.py

```diff
@@ -273,6 +280,13 @@ def __init__(self) -> None:
                 replacement_topic="event-replacements",
                 commit_log_topic="snuba-commit-log",
             ),
+            replacer_processor=ErrorsReplacer(
```

Comment:
This way the replacer becomes defined at storage level (which means at table level) and not at dataset level anymore.
If one day we will need multi-table replacers we could revisit that, but I guess we should flesh out a realistic requirement for that (which we do not have today).